js/src/Axis.js

```diff
@@ -35,22 +35,15 @@ var Axis = widgets.WidgetView.extend({
 
     render: function() {
         this.d3el.style("display", this.model.get("visible") ? "inline" : "none");
-
         this.parent = this.options.parent;
         this.margin = this.parent.margin;
-        this.vertical = this.model.get("orientation") === "vertical";
         this.height = this.parent.height - (this.margin.top + this.margin.bottom);
         this.width = this.parent.width - (this.margin.left + this.margin.right);
 
-        var scale_promise = this.set_scale(this.model.get("scale"));
-        this.side = this.model.get("side");
-        this.padding = this.model.get("padding");
-        this.num_ticks = this.model.get("num_ticks");
-        this.label_loc = this.model.get("label_location");
-        this.label_offset = this.extract_label_offset(this.model.get("label_offset"));
+        let scale_promise = this.set_scale_promise(this.model.get("scale"));
```

Comment:
Ping @jasongrout. Wonder if we should use babel to allow the use of ES6 syntax in bqplot?

(`const`, `let`).